### ISSUE TITLE
copy-helper.sh script "No such file or directory" when profiling

### ISSUE THREAD

**Issue by jacobrosenthal** [sindresorhus/LaunchAtLogin#7]
Love the update and simplification, it pushed me to learn and reimplement my app in swift

Its has been working great, but when I try to profile the copy-helper.sh script cant be found.. any ideas?

```
Showing All Issues
PhaseScriptExecution Launch\ At\ Login\ script /Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/Script-5B9CE72B1FD2258700CA8BFE.sh
    cd /Users/jacobrosenthal/Downloads/untitled/memoryio
    export ACTION=build
    export ALTERNATE_GROUP=staff
    export ALTERNATE_MODE=u+w,go-w,a+rX
    export ALTERNATE_OWNER=jacobrosenthal
    export ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES=NO
    export ALWAYS_SEARCH_USER_PATHS=NO
    export ALWAYS_USE_SEPARATE_HEADERMAPS=NO
    export APPLE_INTERNAL_DEVELOPER_DIR=/AppleInternal/Developer
    export APPLE_INTERNAL_DIR=/AppleInternal
    export APPLE_INTERNAL_DOCUMENTATION_DIR=/AppleInternal/Documentation
    export APPLE_INTERNAL_LIBRARY_DIR=/AppleInternal/Library
    export APPLE_INTERNAL_TOOLS=/AppleInternal/Developer/Tools
    export APPLICATION_EXTENSION_API_ONLY=NO
    export APPLY_RULES_IN_COPY_FILES=NO
    export ARCHS=x86_64
    export ARCHS_STANDARD=x86_64
    export ARCHS_STANDARD_32_64_BIT="x86_64 i386"
    export ARCHS_STANDARD_32_BIT=i386
    export ARCHS_STANDARD_64_BIT=x86_64
    export ARCHS_STANDARD_INCLUDING_64_BIT=x86_64
    export ASSETCATALOG_COMPILER_APPICON_NAME=AppIcon
    export AVAILABLE_PLATFORMS="appletvos appletvsimulator iphoneos iphonesimulator macosx watchos watchsimulator"
    export BITCODE_GENERATION_MODE=marker
    export BUILD_ACTIVE_RESOURCES_ONLY=NO
    export BUILD_COMPONENTS="headers build"
    export BUILD_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products
    export BUILD_ROOT=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products
    export BUILD_STYLE=
    export BUILD_VARIANTS=normal
    export BUILT_PRODUCTS_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release
    export CACHE_ROOT=/var/folders/hv/syzcx3w10850txk2z6tbnb9m0000gn/C/com.apple.DeveloperTools/9.2-9C40b/Xcode
    export CCHROOT=/var/folders/hv/syzcx3w10850txk2z6tbnb9m0000gn/C/com.apple.DeveloperTools/9.2-9C40b/Xcode
    export CHMOD=/bin/chmod
    export CHOWN=/usr/sbin/chown
    export CLANG_ANALYZER_NONNULL=YES
    export CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION=YES_AGGRESSIVE
    export CLANG_CXX_LANGUAGE_STANDARD=gnu++14
    export CLANG_CXX_LIBRARY=libc++
    export CLANG_ENABLE_MODULES=YES
    export CLANG_ENABLE_OBJC_ARC=YES
    export CLANG_MODULES_BUILD_SESSION_FILE=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/ModuleCache/Session.modulevalidation
    export CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING=YES
    export CLANG_WARN_BOOL_CONVERSION=YES
    export CLANG_WARN_COMMA=YES
    export CLANG_WARN_CONSTANT_CONVERSION=YES
    export CLANG_WARN_DIRECT_OBJC_ISA_USAGE=YES_ERROR
    export CLANG_WARN_DOCUMENTATION_COMMENTS=YES
    export CLANG_WARN_EMPTY_BODY=YES
    export CLANG_WARN_ENUM_CONVERSION=YES
    export CLANG_WARN_INFINITE_RECURSION=YES
    export CLANG_WARN_INT_CONVERSION=YES
    export CLANG_WARN_NON_LITERAL_NULL_CONVERSION=YES
    export CLANG_WARN_OBJC_LITERAL_CONVERSION=YES
    export CLANG_WARN_OBJC_ROOT_CLASS=YES_ERROR
    export CLANG_WARN_RANGE_LOOP_ANALYSIS=YES
    export CLANG_WARN_STRICT_PROTOTYPES=YES
    export CLANG_WARN_SUSPICIOUS_MOVE=YES
    export CLANG_WARN_UNGUARDED_AVAILABILITY=YES_AGGRESSIVE
    export CLANG_WARN_UNREACHABLE_CODE=YES
    export CLANG_WARN__DUPLICATE_METHOD_MATCH=YES
    export CLASS_FILE_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/JavaClasses
    export CLEAN_PRECOMPS=YES
    export CLONE_HEADERS=NO
    export CODESIGNING_FOLDER_PATH=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release/memoryio.app
    export CODE_SIGNING_ALLOWED=YES
    export CODE_SIGN_ENTITLEMENTS=memoryio/memoryio.entitlements
    export CODE_SIGN_IDENTITY=-
    export CODE_SIGN_STYLE=Manual
    export COLOR_DIAGNOSTICS=NO
    export COMBINE_HIDPI_IMAGES=YES
    export COMMAND_MODE=legacy
    export COMPILER_INDEX_STORE_ENABLE=Default
    export COMPOSITE_SDK_DIRS=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/CompositeSDKs
    export COMPRESS_PNG_FILES=NO
    export CONFIGURATION=Release
    export CONFIGURATION_BUILD_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release
    export CONFIGURATION_TEMP_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release
    export CONTENTS_FOLDER_PATH=memoryio.app/Contents
    export COPYING_PRESERVES_HFS_DATA=NO
    export COPY_HEADERS_RUN_UNIFDEF=NO
    export COPY_PHASE_STRIP=NO
    export COPY_RESOURCES_FROM_STATIC_FRAMEWORKS=YES
    export CP=/bin/cp
    export CREATE_INFOPLIST_SECTION_IN_BINARY=NO
    export CURRENT_ARCH=x86_64
    export CURRENT_VARIANT=normal
    export DEAD_CODE_STRIPPING=NO
    export DEBUGGING_SYMBOLS=YES
    export DEBUG_INFORMATION_FORMAT=dwarf-with-dsym
    export DEFAULT_COMPILER=com.apple.compilers.llvm.clang.1_0
    export DEFAULT_KEXT_INSTALL_PATH=/Library/Extensions
    export DEFINES_MODULE=NO
    export DEPLOYMENT_LOCATION=NO
    export DEPLOYMENT_POSTPROCESSING=NO
    export DEPLOYMENT_TARGET_CLANG_ENV_NAME=MACOSX_DEPLOYMENT_TARGET
    export DEPLOYMENT_TARGET_CLANG_FLAG_NAME=mmacosx-version-min
    export DEPLOYMENT_TARGET_SETTING_NAME=MACOSX_DEPLOYMENT_TARGET
    export DERIVED_FILES_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/DerivedSources
    export DERIVED_FILE_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/DerivedSources
    export DERIVED_SOURCES_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/DerivedSources
    export DEVELOPER_APPLICATIONS_DIR=/Applications/Xcode.app/Contents/Developer/Applications
    export DEVELOPER_BIN_DIR=/Applications/Xcode.app/Contents/Developer/usr/bin
    export DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer
    export DEVELOPER_FRAMEWORKS_DIR=/Applications/Xcode.app/Contents/Developer/Library/Frameworks
    export DEVELOPER_FRAMEWORKS_DIR_QUOTED=/Applications/Xcode.app/Contents/Developer/Library/Frameworks
    export DEVELOPER_LIBRARY_DIR=/Applications/Xcode.app/Contents/Developer/Library
    export DEVELOPER_SDK_DIR=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs
    export DEVELOPER_TOOLS_DIR=/Applications/Xcode.app/Contents/Developer/Tools
    export DEVELOPER_USR_DIR=/Applications/Xcode.app/Contents/Developer/usr
    export DEVELOPMENT_LANGUAGE=en
    export DOCUMENTATION_FOLDER_PATH=memoryio.app/Contents/Resources/en.lproj/Documentation
    export DO_HEADER_SCANNING_IN_JAM=NO
    export DSTROOT=/tmp/memoryio.dst
    export DT_TOOLCHAIN_DIR=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain
    export DWARF_DSYM_FILE_NAME=memoryio.app.dSYM
    export DWARF_DSYM_FILE_SHOULD_ACCOMPANY_PRODUCT=NO
    export DWARF_DSYM_FOLDER_PATH=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release
    export EMBEDDED_CONTENT_CONTAINS_SWIFT=NO
    export EMBEDDED_PROFILE_NAME=embedded.provisionprofile
    export EMBED_ASSET_PACKS_IN_PRODUCT_BUNDLE=NO
    export ENABLE_BITCODE=NO
    export ENABLE_DEFAULT_HEADER_SEARCH_PATHS=YES
    export ENABLE_HEADER_DEPENDENCIES=YES
    export ENABLE_NS_ASSERTIONS=NO
    export ENABLE_ON_DEMAND_RESOURCES=NO
    export ENABLE_STRICT_OBJC_MSGSEND=YES
    export ENABLE_TESTABILITY=NO
    export EXCLUDED_INSTALLSRC_SUBDIRECTORY_PATTERNS=".DS_Store .svn .git .hg CVS"
    export EXCLUDED_RECURSIVE_SEARCH_PATH_SUBDIRECTORIES="*.nib *.lproj *.framework *.gch *.xcode* *.xcassets (*) .DS_Store CVS .svn .git .hg *.pbproj *.pbxproj"
    export EXECUTABLES_FOLDER_PATH=memoryio.app/Contents/Executables
    export EXECUTABLE_FOLDER_PATH=memoryio.app/Contents/MacOS
    export EXECUTABLE_NAME=memoryio
    export EXECUTABLE_PATH=memoryio.app/Contents/MacOS/memoryio
    export EXPANDED_CODE_SIGN_IDENTITY=-
    export EXPANDED_CODE_SIGN_IDENTITY_NAME=-
    export EXPANDED_PROVISIONING_PROFILE=
    export FILE_LIST=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/Objects/LinkFileList
    export FIXED_FILES_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/FixedFiles
    export FRAMEWORKS_FOLDER_PATH=memoryio.app/Contents/Frameworks
    export FRAMEWORK_FLAG_PREFIX=-framework
    export FRAMEWORK_SEARCH_PATHS="/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release  /Users/jacobrosenthal/Downloads/untitled/memoryio/Carthage/Build/Mac"
    export FRAMEWORK_VERSION=A
    export FULL_PRODUCT_NAME=memoryio.app
    export GCC3_VERSION=3.3
    export GCC_C_LANGUAGE_STANDARD=gnu11
    export GCC_DYNAMIC_NO_PIC=NO
    export GCC_INLINES_ARE_PRIVATE_EXTERN=YES
    export GCC_NO_COMMON_BLOCKS=YES
    export GCC_PFE_FILE_C_DIALECTS="c objective-c c++ objective-c++"
    export GCC_SYMBOLS_PRIVATE_EXTERN=YES
    export GCC_TREAT_WARNINGS_AS_ERRORS=NO
    export GCC_VERSION=com.apple.compilers.llvm.clang.1_0
    export GCC_VERSION_IDENTIFIER=com_apple_compilers_llvm_clang_1_0
    export GCC_WARN_64_TO_32_BIT_CONVERSION=YES
    export GCC_WARN_ABOUT_RETURN_TYPE=YES_ERROR
    export GCC_WARN_UNDECLARED_SELECTOR=YES
    export GCC_WARN_UNINITIALIZED_AUTOS=YES_AGGRESSIVE
    export GCC_WARN_UNUSED_FUNCTION=YES
    export GCC_WARN_UNUSED_VARIABLE=YES
    export GENERATE_MASTER_OBJECT_FILE=NO
    export GENERATE_PKGINFO_FILE=YES
    export GENERATE_PROFILING_CODE=NO
    export GENERATE_TEXT_BASED_STUBS=NO
    export GID=20
    export GROUP=staff
    export HEADERMAP_INCLUDES_FLAT_ENTRIES_FOR_TARGET_BEING_BUILT=YES
    export HEADERMAP_INCLUDES_FRAMEWORK_ENTRIES_FOR_ALL_PRODUCT_TYPES=YES
    export HEADERMAP_INCLUDES_NONPUBLIC_NONPRIVATE_HEADERS=YES
    export HEADERMAP_INCLUDES_PROJECT_HEADERS=YES
    export HEADERMAP_USES_FRAMEWORK_PREFIX_ENTRIES=YES
    export HEADERMAP_USES_VFS=NO
    export HEADER_SEARCH_PATHS="/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release/include "
    export HIDE_BITCODE_SYMBOLS=YES
    export HOME=/Users/jacobrosenthal
    export ICONV=/usr/bin/iconv
    export INFOPLIST_EXPAND_BUILD_SETTINGS=YES
    export INFOPLIST_FILE=memoryio/Info.plist
    export INFOPLIST_OUTPUT_FORMAT=same-as-input
    export INFOPLIST_PATH=memoryio.app/Contents/Info.plist
    export INFOPLIST_PREPROCESS=NO
    export INFOSTRINGS_PATH=memoryio.app/Contents/Resources/en.lproj/InfoPlist.strings
    export INLINE_PRIVATE_FRAMEWORKS=NO
    export INSTALLHDRS_COPY_PHASE=NO
    export INSTALLHDRS_SCRIPT_PHASE=NO
    export INSTALL_DIR=/tmp/memoryio.dst/Applications
    export INSTALL_GROUP=staff
    export INSTALL_MODE_FLAG=u+w,go-w,a+rX
    export INSTALL_OWNER=jacobrosenthal
    export INSTALL_PATH=/Applications
    export INSTALL_ROOT=/tmp/memoryio.dst
    export JAVAC_DEFAULT_FLAGS="-J-Xms64m -J-XX:NewSize=4M -J-Dfile.encoding=UTF8"
    export JAVA_APP_STUB=/System/Library/Frameworks/JavaVM.framework/Resources/MacOS/JavaApplicationStub
    export JAVA_ARCHIVE_CLASSES=YES
    export JAVA_ARCHIVE_TYPE=JAR
    export JAVA_COMPILER=/usr/bin/javac
    export JAVA_FOLDER_PATH=memoryio.app/Contents/Resources/Java
    export JAVA_FRAMEWORK_RESOURCES_DIRS=Resources
    export JAVA_JAR_FLAGS=cv
    export JAVA_SOURCE_SUBDIR=.
    export JAVA_USE_DEPENDENCIES=YES
    export JAVA_ZIP_FLAGS=-urg
    export JIKES_DEFAULT_FLAGS="+E +OLDCSO"
    export KASAN_DEFAULT_CFLAGS="-DKASAN=1 -fsanitize=address -mllvm -asan-globals-live-support -mllvm -asan-force-dynamic-shadow"
    export KEEP_PRIVATE_EXTERNS=NO
    export LD_DEPENDENCY_INFO_FILE=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/Objects-normal/x86_64/memoryio_dependency_info.dat
    export LD_GENERATE_MAP_FILE=NO
    export LD_MAP_FILE_PATH=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/memoryio-LinkMap-normal-x86_64.txt
    export LD_NO_PIE=NO
    export LD_QUOTE_LINKER_ARGUMENTS_FOR_COMPILER_DRIVER=YES
    export LD_RUNPATH_SEARCH_PATHS=" @executable_path/../Frameworks"
    export LEGACY_DEVELOPER_DIR=/Applications/Xcode.app/Contents/PlugIns/Xcode3Core.ideplugin/Contents/SharedSupport/Developer
    export LEX=lex
    export LIBRARY_FLAG_NOSPACE=YES
    export LIBRARY_FLAG_PREFIX=-l
    export LIBRARY_KEXT_INSTALL_PATH=/Library/Extensions
    export LIBRARY_SEARCH_PATHS="/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release "
    export LINKER_DISPLAYS_MANGLED_NAMES=NO
    export LINK_FILE_LIST_normal_x86_64=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/Objects-normal/x86_64/memoryio.LinkFileList
    export LINK_WITH_STANDARD_LIBRARIES=YES
    export LOCALIZABLE_CONTENT_DIR=
    export LOCALIZED_RESOURCES_FOLDER_PATH=memoryio.app/Contents/Resources/en.lproj
    export LOCAL_ADMIN_APPS_DIR=/Applications/Utilities
    export LOCAL_APPS_DIR=/Applications
    export LOCAL_DEVELOPER_DIR=/Library/Developer
    export LOCAL_LIBRARY_DIR=/Library
    export LOCROOT=
    export LOCSYMROOT=
    export MACH_O_TYPE=mh_execute
    export MACOSX_DEPLOYMENT_TARGET=10.13
    export MAC_OS_X_PRODUCT_BUILD_VERSION=17C88
    export MAC_OS_X_VERSION_ACTUAL=101302
    export MAC_OS_X_VERSION_MAJOR=101300
    export MAC_OS_X_VERSION_MINOR=1302
    export METAL_LIBRARY_FILE_BASE=default
    export METAL_LIBRARY_OUTPUT_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release/memoryio.app/Contents/Resources
    export MODULE_CACHE_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/ModuleCache
    export MTL_ENABLE_DEBUG_INFO=NO
    export NATIVE_ARCH=i386
    export NATIVE_ARCH_32_BIT=i386
    export NATIVE_ARCH_64_BIT=x86_64
    export NATIVE_ARCH_ACTUAL=x86_64
    export NO_COMMON=YES
    export OBJECT_FILE_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/Objects
    export OBJECT_FILE_DIR_normal=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/Objects-normal
    export OBJROOT=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex
    export ONLY_ACTIVE_ARCH=NO
    export OS=MACOS
    export OSAC=/usr/bin/osacompile
    export PACKAGE_TYPE=com.apple.package-type.wrapper.application
    export PASCAL_STRINGS=YES
    export PATH="/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin:/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/local/bin:/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/libexec:/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/usr/local/bin:/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/usr/bin:/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/usr/local/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/local/bin:/Applications/Xcode.app/Contents/Developer/Tools:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    export PATH_PREFIXES_EXCLUDED_FROM_HEADER_DEPENDENCIES="/usr/include /usr/local/include /System/Library/Frameworks /System/Library/PrivateFrameworks /Applications/Xcode.app/Contents/Developer/Headers /Applications/Xcode.app/Contents/Developer/SDKs /Applications/Xcode.app/Contents/Developer/Platforms"
    export PBDEVELOPMENTPLIST_PATH=memoryio.app/Contents/pbdevelopment.plist
    export PKGINFO_FILE_PATH=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/PkgInfo
    export PKGINFO_PATH=memoryio.app/Contents/PkgInfo
    export PLATFORM_DEVELOPER_APPLICATIONS_DIR=/Applications/Xcode.app/Contents/Developer/Applications
    export PLATFORM_DEVELOPER_BIN_DIR=/Applications/Xcode.app/Contents/Developer/usr/bin
    export PLATFORM_DEVELOPER_LIBRARY_DIR=/Applications/Xcode.app/Contents/Developer/Library
    export PLATFORM_DEVELOPER_SDK_DIR=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs
    export PLATFORM_DEVELOPER_TOOLS_DIR=/Applications/Xcode.app/Contents/Developer/Tools
    export PLATFORM_DEVELOPER_USR_DIR=/Applications/Xcode.app/Contents/Developer/usr
    export PLATFORM_DIR=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform
    export PLATFORM_DISPLAY_NAME=macOS
    export PLATFORM_NAME=macosx
    export PLATFORM_PREFERRED_ARCH=x86_64
    export PLATFORM_PRODUCT_BUILD_VERSION=9C40b
    export PLIST_FILE_OUTPUT_FORMAT=same-as-input
    export PLUGINS_FOLDER_PATH=memoryio.app/Contents/PlugIns
    export PRECOMPS_INCLUDE_HEADERS_FROM_BUILT_PRODUCTS_DIR=YES
    export PRECOMP_DESTINATION_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/PrefixHeaders
    export PRESERVE_DEAD_CODE_INITS_AND_TERMS=NO
    export PRIVATE_HEADERS_FOLDER_PATH=memoryio.app/Contents/PrivateHeaders
    export PRODUCT_BUNDLE_IDENTIFIER=com.augmentous.memoryio
    export PRODUCT_MODULE_NAME=memoryio
    export PRODUCT_NAME=memoryio
    export PRODUCT_SETTINGS_PATH=/Users/jacobrosenthal/Downloads/untitled/memoryio/memoryio/Info.plist
    export PRODUCT_TYPE=com.apple.product-type.application
    export PROFILING_CODE=NO
    export PROJECT=memoryio
    export PROJECT_DERIVED_FILE_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/DerivedSources
    export PROJECT_DIR=/Users/jacobrosenthal/Downloads/untitled/memoryio
    export PROJECT_FILE_PATH=/Users/jacobrosenthal/Downloads/untitled/memoryio/memoryio.xcodeproj
    export PROJECT_NAME=memoryio
    export PROJECT_TEMP_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build
    export PROJECT_TEMP_ROOT=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex
    export PUBLIC_HEADERS_FOLDER_PATH=memoryio.app/Contents/Headers
    export RECURSIVE_SEARCH_PATHS_FOLLOW_SYMLINKS=YES
    export REMOVE_CVS_FROM_RESOURCES=YES
    export REMOVE_GIT_FROM_RESOURCES=YES
    export REMOVE_HEADERS_FROM_EMBEDDED_BUNDLES=YES
    export REMOVE_HG_FROM_RESOURCES=YES
    export REMOVE_SVN_FROM_RESOURCES=YES
    export REZ_COLLECTOR_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/ResourceManagerResources
    export REZ_OBJECTS_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/ResourceManagerResources/Objects
    export REZ_SEARCH_PATHS="/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release "
    export SCAN_ALL_SOURCE_FILES_FOR_INCLUDES=NO
    export SCRIPTS_FOLDER_PATH=memoryio.app/Contents/Resources/Scripts
    export SCRIPT_INPUT_FILE_COUNT=0
    export SCRIPT_OUTPUT_FILE_COUNT=0
    export SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk
    export SDK_DIR=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk
    export SDK_DIR_macosx10_13=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk
    export SDK_NAME=macosx10.13
    export SDK_NAMES=macosx10.13
    export SDK_PRODUCT_BUILD_VERSION=17C76
    export SDK_VERSION=10.13
    export SDK_VERSION_ACTUAL=101300
    export SDK_VERSION_MAJOR=101300
    export SDK_VERSION_MINOR=1300
    export SED=/usr/bin/sed
    export SEPARATE_STRIP=NO
    export SEPARATE_SYMBOL_EDIT=NO
    export SET_DIR_MODE_OWNER_GROUP=YES
    export SET_FILE_MODE_OWNER_GROUP=NO
    export SHALLOW_BUNDLE=NO
    export SHARED_DERIVED_FILE_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release/DerivedSources
    export SHARED_FRAMEWORKS_FOLDER_PATH=memoryio.app/Contents/SharedFrameworks
    export SHARED_PRECOMPS_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/PrecompiledHeaders
    export SHARED_SUPPORT_FOLDER_PATH=memoryio.app/Contents/SharedSupport
    export SKIP_INSTALL=NO
    export SOURCE_ROOT=/Users/jacobrosenthal/Downloads/untitled/memoryio
    export SRCROOT=/Users/jacobrosenthal/Downloads/untitled/memoryio
    export STRINGS_FILE_OUTPUT_ENCODING=UTF-16
    export STRIP_BITCODE_FROM_COPIED_FILES=NO
    export STRIP_INSTALLED_PRODUCT=YES
    export STRIP_PNG_TEXT=NO
    export STRIP_STYLE=all
    export STRIP_SWIFT_SYMBOLS=YES
    export SUPPORTED_PLATFORMS=macosx
    export SUPPORTS_TEXT_BASED_API=NO
    export SWIFT_OBJC_BRIDGING_HEADER=memoryio/memoryio-Bridging-Header.h
    export SWIFT_OPTIMIZATION_LEVEL=-Owholemodule
    export SWIFT_PLATFORM_TARGET_PREFIX=macosx
    export SWIFT_VERSION=4.0
    export SYMROOT=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products
    export SYSTEM_ADMIN_APPS_DIR=/Applications/Utilities
    export SYSTEM_APPS_DIR=/Applications
    export SYSTEM_CORE_SERVICES_DIR=/System/Library/CoreServices
    export SYSTEM_DEMOS_DIR=/Applications/Extras
    export SYSTEM_DEVELOPER_APPS_DIR=/Applications/Xcode.app/Contents/Developer/Applications
    export SYSTEM_DEVELOPER_BIN_DIR=/Applications/Xcode.app/Contents/Developer/usr/bin
    export SYSTEM_DEVELOPER_DEMOS_DIR="/Applications/Xcode.app/Contents/Developer/Applications/Utilities/Built Examples"
    export SYSTEM_DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer
    export SYSTEM_DEVELOPER_DOC_DIR="/Applications/Xcode.app/Contents/Developer/ADC Reference Library"
    export SYSTEM_DEVELOPER_GRAPHICS_TOOLS_DIR="/Applications/Xcode.app/Contents/Developer/Applications/Graphics Tools"
    export SYSTEM_DEVELOPER_JAVA_TOOLS_DIR="/Applications/Xcode.app/Contents/Developer/Applications/Java Tools"
    export SYSTEM_DEVELOPER_PERFORMANCE_TOOLS_DIR="/Applications/Xcode.app/Contents/Developer/Applications/Performance Tools"
    export SYSTEM_DEVELOPER_RELEASENOTES_DIR="/Applications/Xcode.app/Contents/Developer/ADC Reference Library/releasenotes"
    export SYSTEM_DEVELOPER_TOOLS=/Applications/Xcode.app/Contents/Developer/Tools
    export SYSTEM_DEVELOPER_TOOLS_DOC_DIR="/Applications/Xcode.app/Contents/Developer/ADC Reference Library/documentation/DeveloperTools"
    export SYSTEM_DEVELOPER_TOOLS_RELEASENOTES_DIR="/Applications/Xcode.app/Contents/Developer/ADC Reference Library/releasenotes/DeveloperTools"
    export SYSTEM_DEVELOPER_USR_DIR=/Applications/Xcode.app/Contents/Developer/usr
    export SYSTEM_DEVELOPER_UTILITIES_DIR=/Applications/Xcode.app/Contents/Developer/Applications/Utilities
    export SYSTEM_DOCUMENTATION_DIR=/Library/Documentation
    export SYSTEM_KEXT_INSTALL_PATH=/System/Library/Extensions
    export SYSTEM_LIBRARY_DIR=/System/Library
    export TAPI_VERIFY_MODE=ErrorsOnly
    export TARGETNAME=memoryio
    export TARGET_BUILD_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release
    export TARGET_NAME=memoryio
    export TARGET_TEMP_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build
    export TEMP_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build
    export TEMP_FILES_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build
    export TEMP_FILE_DIR=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build
    export TEMP_ROOT=/Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex
    export TOOLCHAINS=com.apple.dt.toolchain.XcodeDefault
    export TOOLCHAIN_DIR=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain
    export TREAT_MISSING_BASELINES_AS_TEST_FAILURES=NO
    export UID=501
    export UNLOCALIZED_RESOURCES_FOLDER_PATH=memoryio.app/Contents/Resources
    export UNSTRIPPED_PRODUCT=NO
    export USER=jacobrosenthal
    export USER_APPS_DIR=/Users/jacobrosenthal/Applications
    export USER_LIBRARY_DIR=/Users/jacobrosenthal/Library
    export USE_DYNAMIC_NO_PIC=YES
    export USE_HEADERMAP=YES
    export USE_HEADER_SYMLINKS=NO
    export VALIDATE_PRODUCT=NO
    export VALID_ARCHS="i386 x86_64"
    export VERBOSE_PBXCP=NO
    export VERSIONPLIST_PATH=memoryio.app/Contents/version.plist
    export VERSION_INFO_BUILDER=jacobrosenthal
    export VERSION_INFO_FILE=memoryio_vers.c
    export VERSION_INFO_STRING="\"@(#)PROGRAM:memoryio  PROJECT:memoryio-\""
    export WRAPPER_EXTENSION=app
    export WRAPPER_NAME=memoryio.app
    export WRAPPER_SUFFIX=.app
    export WRAP_ASSET_PACKS_IN_SEPARATE_DIRECTORIES=NO
    export XCODE_APP_SUPPORT_DIR=/Applications/Xcode.app/Contents/Developer/Library/Xcode
    export XCODE_PRODUCT_BUILD_VERSION=9C40b
    export XCODE_VERSION_ACTUAL=0920
    export XCODE_VERSION_MAJOR=0900
    export XCODE_VERSION_MINOR=0920
    export XPCSERVICES_FOLDER_PATH=memoryio.app/Contents/XPCServices
    export YACC=yacc
    export arch=x86_64
    export variant=normal
    /bin/sh -c /Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Intermediates.noindex/memoryio.build/Release/memoryio.build/Script-5B9CE72B1FD2258700CA8BFE.sh

cp: /Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release/memoryio.app/Contents/Frameworks/LaunchAtLogin.framework/Resources/LaunchAtLoginHelper.app: No such file or directory
rm: /Users/jacobrosenthal/Library/Developer/Xcode/DerivedData/memoryio-dlcrfdgchgsogtdsjkkyjanigonw/Build/Products/Release/memoryio.app/Contents/Frameworks/LaunchAtLogin.framework/Resources/copy-helper.sh: No such file or directory
```

**Comment by sindresorhus** [comment]
I just tried to profile an app I have that uses this module and it worked fine.

Maybe try `carthage build` again.

**Comment by pjloury** [comment]
I'm also getting 
"rm: /copy-helper.sh: No such file or directory" when archiving my project.
@jacobrosenthal were you able to fix your issue?

**Comment by jacobrosenthal** [comment]
Yeah it was weird, I think `carthage build` fixes it, but I had to do that each time I profiled.

**Comment by sindresorhus** [comment]
> I was getting other errors that I was able to fix by dragging the LaunchAtLogin.framework into the "Embedded Binaries" section in addition to the "Linked Frameworks and Libraries" section under Target > General

That's already covered by the Carthage guide:

> On your application targets’ General settings tab, in the Embedded Binaries section, drag and drop each framework you want to use from the Carthage/Build folder on disk.
> \- https://github.com/Carthage/Carthage#getting-started

**Comment by deadcoder0904** [comment]
@pjloury @jacobrosenthal I'm getting this error right now.

```
cp: /Users/deadcoder0904/Library/Developer/Xcode/DerivedData/Away_From_Keyboard-bcrytpktfvzflmexubmyosnhzksq/Build/Products/Debug/Away From Keyboard.app/Contents/Frameworks/LaunchAtLogin.framework/Resources/LaunchAtLoginHelper.app: No such file or directory
Command /bin/sh emitted errors but did not return a nonzero exit code to indicate failure
```

while using Podfile.

Did you use it with Podfile? Or any solution how to fix it?

**Comment by wes566** [comment]
I still get this error, both  on debug and release builds when using cocoapods.

Xcode 9.4.1
Swift 4.1 project

I tried @deadcoder0904 proposed solution but that didn't help.  The script itself runs for me so I wouldn't think there is any problem with the build phase step finding the path to the script.

The real problem seems to be that when I build my workspace the LaunchAtLoginHelper.app isn't being created (like when I open the folder in Finder I see that the .app isn't there).  Is it possible there is some other xcworkspace setting that I need to change to make sure the POD is building LaunchAtLoginHelper.app?

Also, @sindresorhus do you happen to have a sample repo that consumes this via cocoapods as a reference?

Thanks!

**Comment by deadcoder0904** [comment]
@wes566 I happen to have made both examples with a simple project :)

CocoaPods Example - https://github.com/deadcoder0904/LaunchAtLoginPod

Carthage Example - https://github.com/deadcoder0904/LaunchAtLoginCart

**Comment by wes566** [comment]
Thanks @deadcoder0904 - I tried your repo but the script failed to run after a fresh clone, so that's why I was asking @sindresorhus if there was another repo I could look at as a reference.

**Comment by IssuehuntBot** [comment]
@issuehuntfest has funded $80.00 to this issue. [See it on IssueHunt](https://issuehunt.io/repos/98812730/issues/7)

**Comment by Bargsteen** [comment]
I just updated XCode and it installed Swift 5.0, which forced me to update my Carthage packages; and now I get this same exact error. Any updates as to what might be the problem?

**Comment by nodeful** [comment]
I came across the same issue when building my app in Release mode. 
First build after doing `carthage build` builds fine but all consecutive builds don't. 
Seems like during the release build the copy-helper.sh and LaunchAtLoginHelper.app are being deleted:
![image](https://user-images.githubusercontent.com/8472525/71886853-d5029c80-3134-11ea-984e-d75d8b0dda95.png)
By commenting out these lines it seems to work:
![image](https://user-images.githubusercontent.com/8472525/71886949-fd8a9680-3134-11ea-9890-6175f9aedf1a.png)

**Comment by sindresorhus** [comment]
@nodeful That removes the helper app and the copy script from the framework embedded in your app as you already have the helper app elsewhere in the bundle and the copy script is not useful inside your built app.

**Comment by sweetleon** [comment]
@sindresorhus Have you had a chance to look at #34 as a solution to this issue?

**Comment by sindresorhus** [comment]
I will look at it soon. I’m on vacation at the moment.

**Comment by sweetleon** [comment]
@sindresorhus have you had a chance to look at #34 ?

**Comment by sindresorhus** [comment]
@sweetleon I have already reviewed it.

**Comment by nodeful** [comment]
I did try @sweetleon #34 PR code and it does seem to fix the issue for me, not sure if the changes have other implications however, didn't see any so far.

**Comment by nodeful** [comment]
☝️ actually Archiving has stopped working after using code in #34

**Comment by frogg** [comment]
@nodeful make sure that you put the "Run Script" area all the way to the bottom. I had it at a higher position before which caused the problems for me.
![image](https://user-images.githubusercontent.com/5204169/81193431-ad74f100-8fbb-11ea-9e8c-0722670b3190.png)

**Comment by nodeful** [comment]
it was last already @frogg 
In the end I've just did this: https://github.com/nodeful/LaunchAtLogin/commit/1fac49ee0d691da94c4ade78a503fa6d480477d1
and it's seem to work in all cases.
I think the problem could be related to different Xcode project behaviours. In my case the Embed Frameworks or the Copy Frameworks (carthage) step would move the frameworks into the bundle only once on initial build or after a clean but never after that, where's for some it could be copying everything every time.

**Comment by rose-m** [comment]
I can confirm the following behavior:

* When running with _Debug_ as Build Configuration it always works
* When running with _Release_ as Build Configuration, it _only works_ the **first time or after a clean of the project**. If I do not clean and just try to re-run I get the following error:

```
cp: .../yFi-gljjlugsyuqykqbjhjsullowoqbk/Build/Products/Release/yFi.app/Contents/Frameworks/LaunchAtLogin.framework/Resources/LaunchAtLoginHelper.app: No such file or directory
.../yFi-gljjlugsyuqykqbjhjsullowoqbk/Build/Products/Release/yFi.app/Contents/Frameworks/LaunchAtLogin.framework/Resources/LaunchAtLogin.entitlements: cannot read entitlement data
rm: .../yFi-gljjlugsyuqykqbjhjsullowoqbk/Build/Products/Release/yFi.app/Contents/Frameworks/LaunchAtLogin.framework/Resources/copy-helper.sh: No such file or directory
rm: .../yFi-gljjlugsyuqykqbjhjsullowoqbk/Build/Products/Release/yFi.app/Contents/Frameworks/LaunchAtLogin.framework/Resources/LaunchAtLogin.entitlements: No such file or directory
Command PhaseScriptExecution failed with a nonzero exit code
```

The project configuration is correct - the _Run Script_ phase is _after_ the _Embed Frameworks_ phase.

**Comment by sindresorhus** [comment]
I'd recommend switching to SPM for this, which should have fewer problems: https://github.com/sindresorhus/LaunchAtLogin/releases/tag/v4.0.0

**Comment by caiwenshuo** [comment]
I found that if your path(or your project name) contains **spaces**, "No such file or directory" problem will occur when archive.
The solution is to wrap "BUILT_PRODUCTS_DIR" in quotes:
"${BUILT_PRODUCTS_DIR}"/LaunchAtLogin_LaunchAtLogin.bundle/Contents/Resources/copy-helper-swiftpm.sh

**Comment by sindresorhus** [comment]
@caiwenshuo It's already wrapped in quotes in the readme:

```sh
"${BUILT_PRODUCTS_DIR}/LaunchAtLogin_LaunchAtLogin.bundle/Contents/Resources/copy-helper-swiftpm.sh"
```

**Comment by sindresorhus** [comment]
Closing as I don't plan more work on this package. I recommend moving to https://github.com/sindresorhus/LaunchAtLogin-Modern
